tests/test_teacher.py

```diff
@@ -128,8 +128,8 @@ def test_PathToStateQuery():
     assert isinstance(response, PathToStateResponse)
     assert response.query is query
     assert isinstance(response.teacher_traj, LowLevelTrajectory)
```

Comment:
how did the traj lengths change?

Reply:
due to snapping, one fewer step is required to pick/place